platform.md

```diff
@@ -544,7 +542,6 @@ Usage:
 | `<process-type>`    | `CNB_PROCESS_TYPE`         |                     | Default process type to set in the exported image
 | `<project-metadata>`| `CNB_PROJECT_METADATA_PATH`| `<layers>/project-metadata.toml` | Path to a project metadata file (see [`project-metadata.toml`](#project-metadatatoml-toml)
 | `<report>`          | `CNB_REPORT_PATH`          | `<layers>/report.toml`    | Path to report (see [`report.toml`](#reporttoml-toml)
-| `<run-image>`       | `CNB_RUN_IMAGE`            | resolved from `<stack>`   | Run image reference
```

Comment:
I can't comment on the line directly because it wasn't changed, but line 554 should be removed. Instead we should describe how the exporter must read the location of `<run-image>` from `<analyzed>` so that reference to `<run-image>` in the outputs section make sense.